core_data_modules/cleaners/text_cleaner.py

```diff
@@ -0,0 +1,61 @@
+# coding=utf-8
+import string
+
+import six
+
+
+class TextCleaner(object):
```

Comment:
TextCleaner seems rather generic - is there a more precise name we can give this?